keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -612,24 +605,27 @@ message ProvisionerGetInfoResponse {
 This call is made to create the bucket in the backend. If a bucket that matches both name and parameters already exists, then OK (success) must be returned. If a bucket by same name, but different parameters is provided, then the appropriate error code `ALREADY_EXISTS` must be returned by the provisioner. The call to *ProvisonerCreateBucket* MUST be idempotent.
 
 ```
-message ProvisionerCreateBucketRequest {    
-    // This field is REQUIRED
-    string bucket_name = 1;
-
-    map<string,string> bucket_context = 2;
+message ProvisionerCreateBucketRequest {
+    map<string,string> bucket_context = 1;
 
     enum AnonymousBucketAccessMode {
 	PRIVATE = 0;
 	PUBLIC_READ_ONLY = 1;
 	PUBLIC_WRITE_ONLY = 2;
 	PUBLIC_READ_WRITE = 3;
     }
-    
-    AnonymousBucketAccessMode anonymous_bucket_access_mode = 3;
+
+    AnonymousBucketAccessMode anonymous_bucket_access_mode = 2;
 }
 
 message ProvisionerCreateBucketResponse {
-    // Intentionally left blank
+    // This field is REQUIRED
+    string bucket_name = 1;
+
+    // This field is OPTIONAL
+    string bucket_endpoint = 2;
+
+    map<string,string> protocol_parameters = 3;
```

Comment:
Actually we should have `bucket_context` in CreateBucketResponse, similar to `volume_context` in CreateVolumeResponse in CSI spec.

Reply:
That is what protocol_parameters is intended to be.  The data coming back from the driver is basically all protocol struct related data.  This data gets copied into b.protocol.parameters.  We don't see anything that the driver would return that would be set anywhere else.

Reply:
@xing-yang Is ^^ ok with you?